frontend/src/components/profile/info/badgeHandler.ts

```diff
@@ -240,6 +276,22 @@ function getEligibleBadgeInfo(user: User, badge: BadgeType): Badge | undefined {
     return currentBadge;
 }
 
+/**
+ * Gets the cohort badge
+ * @param cohort the cohort
+ * @returns the badge
+ */
+export function getCohortBadge(cohort: string): Badge {
+    const cohortBadge: Badge = {
+        image: getCohortBadgeImage(cohort),
+        title: getCohortBadgeTitle(cohort),
+        message: getCohortBadgeMessage(cohort),
+        glowHexcode: getCohortBadgeGlow(cohort),
+    };
```

Comment:
Honestly none of these functions seem particularly useful, and we could just inline it all here in this object